servers/services/src/main/java/org/projectnessie/services/spi/ContentService.java

```diff
@@ -48,9 +65,24 @@ ContentResponse getContent(
               regexp = HASH_OR_RELATIVE_COMMIT_SPEC_REGEX,
               message = HASH_OR_RELATIVE_COMMIT_SPEC_MESSAGE)
           String hashOnRef,
-      boolean withDocumentation)
+      boolean withDocumentation,
+      boolean forWrite)
       throws NessieNotFoundException;
```

Comment:
Maybe add `throws AccessCheckException` in places like this one?

Reply:
Hm - maybe later. It's a runtime exception though.